### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by yichengt900** [NOAA-CEFI-Regional-Ocean-Modeling/.github#1] [Announcements]
Hi new members! Welcome to the CEFI regional-mom6 modeling sandbox! At this point we have  two private repos: [regional-mom6-tools](https://github.com/NOAA-CEFI-Regional-Ocean-Modeling/regional-mom6-tools), [regional-mom6-workflow](https://github.com/NOAA-CEFI-Regional-Ocean-Modeling/regional-mom6-workflow), as well as 1 public repo: [cobalt](https://github.com/NOAA-CEFI-Regional-Ocean-Modeling/ocean_BGCl) Feel free to play around and send any feedback through discussions to @yichengt900!

**Comment by theresa-cordero** [comment]
Hello all! Thanks for setting this up Yi-Cheng!

**Comment by charliestock** [comment]
Thanks for the invite and setting this up Yi-Cheng....will start sending questions soon, so thanks in advance for your patience!

**Comment by chiaweh2** [comment]
@yichengt900 this is Chia-Wei Hsu from the data portal team. We were discussing whether there should be a centralized place for the CEFI related repo during the data portal meeting today. We currently setup a https://github.com/NOAA-CEFI-Portal organization account since our repos are getting more and more and want a centralized location. But please let us know if a more centralized CEFI organization is planning or is already somewhere so we can transfer the repo to that place after. Thank you!

**Comment by yichengt900** [comment]
Hi @chiaweh2, thanks for bringing this up! We totally agree that having a centralized location for all CEFI data-portal-related repositories is a great idea. Currently, we use the [GFDL CEFI-regional-MOM6](https://github.com/NOAA-GFDL/CEFI-regional-MOM6) repository as an umbrella repo for modeling system components, including model source codes, pre-processing, and post-processing. From my perspective, I think the post-processing part would eventually benefit from this centralized portal. We're open to discussing how to synchronize this aspect once the CEFI regional MOM6 pre-processing package is more developed.

Thanks again you for your efforts, and we look forward to collaborating.

**Reply by chiaweh2** [reply]
thanks. In this case, we will be maintaining our repo in the new established [GitHub org](https://github.com/NOAA-CEFI-Portal) and looking forward to the unified one later down the road!
